server/database/queries/user/getSuperUserUsers.js

```diff
@@ -0,0 +1,5 @@
+const User = require("../../models/User");
+
+module.exports = function getSuperUserSubUsers(superUserId) {

```

Comment:
function name should match filename (I prefer `getSuperUserSubUsers` - it's clearer than `getSuperUserUsers`

please update across other files too